src/components/molecules/TitleUserLogout.js

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { ActionButton } from "../atoms/ActionButton";
+import { Title } from "../atoms/Title";
+
+/**
+ * This component is for the login button :)
+ */
+export function TitleUserLogout(props) {
```

Comment:
Logout button and welcome message should be added to header not title component ? 
![Screen Shot 2021-01-18 at 11 42 45 AM](https://user-images.githubusercontent.com/26819992/104942308-4a2cc280-5982-11eb-83bd-115cfcb6d89b.png)

Should be here 
![Screen Shot 2021-01-18 at 11 43 23 AM](https://user-images.githubusercontent.com/26819992/104942383-60d31980-5982-11eb-9479-9d293b4b1763.png)

Reply:
that's what I have been asking for the last 4 days. and this is where they said to add it

Reply:
I believe during when we talked about the story we said it would be in the header. In most applications this is where it should be. In any case I wasn't made aware that you did not know this so my apologies if things were not clear as I thought they should have been

Reply:
no need to apologize, I understand your point and I missed when the story was being discussed. thanks